src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,113 @@
+package duke;
+import duke.commands.AddCommand;
```

Comment:
Adding a line break between the package and import statements would help to improve readability.